src/Twig/Extension/DateTimeExtension.php

```diff
@@ -40,13 +50,18 @@ public function __construct(DateTimeHelper $helper)
     public function getFilters()
     {
         return [
-            new TwigFilter('format_date', [$this, 'formatDate'], ['is_safe' => ['html']]),
-            new TwigFilter('format_time', [$this, 'formatTime'], ['is_safe' => ['html']]),
-            new TwigFilter('format_datetime', [$this, 'formatDatetime'], ['is_safe' => ['html']]),
+            new TwigFilter('format_date', [$this, 'formatDate'], ['is_safe' => ['html']]), // NEXT_MAJOR: Remove this line
+            new TwigFilter('sonata_format_date', [$this, 'formatDate'], ['is_safe' => ['html']]),
+            new TwigFilter('format_time', [$this, 'formatTime'], ['is_safe' => ['html']]), // NEXT_MAJOR: Remove this line
+            new TwigFilter('sonata_format_time', [$this, 'formatTime'], ['is_safe' => ['html']]),
+            new TwigFilter('format_datetime', [$this, 'formatDatetime'], ['is_safe' => ['html']]), // NEXT_MAJOR: Remove this line
+            new TwigFilter('sonata_format_datetime', [$this, 'formatDatetime'], ['is_safe' => ['html']]),
```

Comment:
Those should use the new runtime directly

Reply:
Done